guide/deployment/deploying-to-docker.md

```diff
@@ -0,0 +1,154 @@
+# Deploying to Docker
+
+Running projects like your bot with Docker is a great way to get your bot running on a VPS, with the added benefits of being able to quickly spin up your bot on other servers using Docker, use environment variables, simple scaling, auto updating on GitHub pushes, and much more.
+
+1. Get access to your VPS's terminal via SSH or other methods.
+2. Install Docker on your VPS by following the specific instructions for your linux [distribution](https://docs.docker.com/engine/install/#server)
+3. Create a [Dockerfile](#creating-a-dockerfile) or [docker-compose](#creating-a-docker-compose-file) to run your bot.
+4. [Build](#building-your-docker-image) your Docker image.
+5. [Run](#running-your-docker-container) your bot.
+
+## Creating a Dockerfile
+
+A Dockerfile is a file that tells Docker how to build your bot.
```

Comment:
not a concise nor precise description, 'how does docker work & how do I use it' seems out of scope also